pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -1198,18 +1198,23 @@ func (p *Provider) editLabels(
 	tagArgsString := strings.Join(tagArgs, ",")
 	commonArgs := []string{"--project", p.GetProject(), fmt.Sprintf("--labels=%s", tagArgsString)}
 
+	var g errgroup.Group
 	for _, v := range vms {
 		vmArgs := make([]string, len(cmdArgs))
 		copy(vmArgs, cmdArgs)
 
 		vmArgs = append(vmArgs, v.Name, "--zone", v.Zone)
 		vmArgs = append(vmArgs, commonArgs...)
-		cmd := exec.Command("gcloud", vmArgs...)
-		if b, err := cmd.CombinedOutput(); err != nil {
-			return errors.Wrapf(err, "Command: gcloud %s\nOutput: %s", vmArgs, string(b))
-		}
+
+		g.Go(func() error {
```

Comment:
Should we worried about API rate limits?

Reply:
I can limit this by the global flag, but if enough concurrent calls for different clusters are invoked it may still result in high API usage.